pkg/build/controller/build/build_controller.go

```diff
@@ -1975,6 +1975,9 @@ func (bc *BuildController) createBuildRegistriesConfigData(config *configv1.Imag
 			Insecure: registryList{
 				Registries: registriesConfig.InsecureRegistries,
 			},
+			Block: registryList{
+				Registries: registriesConfig.BlockedRegistries,
+			},
```

Comment:
@nalind needs at minimum a unit test - I believe I already have coverage here.

Ideally we should update our e2e suite for blocked registries in origin, too.